Common/Data/Custom/TradingEconomics/TradingEconomicsCalendar.cs

```diff
@@ -185,10 +185,10 @@ public string EventRaw
         public string Ticker { get; set; }
 
         /// <summary>
-        /// Unique symbol used by Trading Economics
+        /// The LEAN symbol
         /// </summary>
-        [JsonProperty(PropertyName = "Symbol")]
-        public string TESymbol { get; set; }
+        [JsonProperty(PropertyName = "Symbol"), JsonConverter(typeof(TradingEconomicsSymbolJsonConverter))]
+        public new Symbol Symbol { get; set; }
```

Comment:
Let's remove the Symbol member hiding. It can potentially be problematic if we upcast to BaseData: https://dotnetfiddle.net/P5gaVC 

The rest of the changes are okay. Let's just make sure to make the `ProcessAPIResponse` method convert the API "Symbol" property to null by doing: `rawData["Symbol"] = null`

Reply:
`((JObject)rawData).Remove("Symbol")` did the trick 😃